tests/carma_bc2gtest.F90

```diff
@@ -45,35 +45,29 @@ subroutine test_coagulation_bc2g()
   integer, parameter    :: nstep        = 72
 
   real(kind=f), parameter   :: dtime  = 600._f
-  real(kind=f), parameter   :: deltax = 100._f
-  real(kind=f), parameter   :: deltay = 100._f
   real(kind=f), parameter   :: deltaz = 100._f
   real(kind=f), parameter   :: zmin   = 0._f
-  
+
   integer, parameter        :: I_BLACKCARBON       = 1
   integer, parameter        :: I_ORGANICCARBON     = 2
 
   type(carma_type), target  :: carma
   type(carma_type), pointer :: carma_ptr
   type(carmastate_type)     :: cstate
   integer                   :: rc = 0
-  
-  real(kind=f), allocatable   :: xc(:,:,:)
```

Comment:
Per comments in the carma base branch, we might want to keep xc and yc as optional and also make lat and lon optional. Then it might be nice to have some test with xc and yc, some with lat and lon, and some with neither.